pages/common/jq.md

```diff
@@ -0,0 +1,23 @@
+# jq
+
+> A lightweight and flexible command-line JSON processor.
+
+- Output JSON file:
+
+`jq '' {{file}}`
```

Comment:
The `''` signifies an empty jq query. I'd say this one is rather uncommon. I find myself doing `cat {{ file }} | jq` _far more often_.